### PR TITLE
feat: Hannahs feature - code review

### PR DESCRIPTION
**Pull request by hannahschwen**
added a heading to hannah.html

### CODE REVIEW

File: hannah.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (hannahschwen) [inline hannah.html:1]: I am neutral about the doctype tag

Comment (Kalink52) [reply hannah.html:1]: i think it's a nice doctype tag

Comment (eapostol) [reply hannah.html:1]: I am now going to look at the the code @hannahschwen

File: hannah.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hannah's Center for Kids Who Can't Read Good</title>
+</head>
+<body>
+  <h1>What is this, a center for ants???</h1>
```
Comment (eapostol) [inline hannah.html:9]: What is the heading? Love it!

**Review by eapostol — APPROVED** [review]
love it! APPROVED.

**Pull request merged**